DEPLOYMENTS.md

```diff
@@ -0,0 +1,51 @@
+## When do ReportStream deployments take place?
+
+ReportStream production deployments take place on Tuesdays and Thursday around 10am EST.
+
+The lower environments, test and staging, are automatically deployed on every merge to `master` (a.k.a. continuous deployment).
+
+## How is ReportStream deployed?
+
+ReportStream is deployed via [this GitHub Action](.github/workflows/release.yml).
+
+The GitHub Action automatically deploys changes in the following branches to the specified environments:
+
+| Branch | Environment |
+|:--|:--|
+| `master` | test, staging |
+| `production` | production |
+
+Code merged to the above branches always go through a PR review.
+
+The deployment process is zero-downtime with a blue/green deployment process used [via Azure Function deployment slots](https://docs.microsoft.com/en-us/azure/azure-functions/functions-deployment-slots). The deployment process is [documented in PR #1318](https://github.com/CDCgov/prime-reportstream/issues/1318).
+
+Smoke tests are run manually after a deployment, but in the near future the will run automatically before directing traffic to the newly deployed code ([work tracked in #656](https://github.com/CDCgov/prime-reportstream/issues/656)).
+
+## What is deployed?
+
+The GitHub Action deploys changes in the following directories:
+
+| Directory | Deployment Process |
+|:--|:--|
+| [/prime-router](/prime-router) | New version released to Azure Functions |
+| [/frontend](/frontend) | Static site served via an Azure Storage Account |
+| [/operations](/operations) | Not deployed. [Coming soon in #578](https://github.com/CDCgov/prime-reportstream/issues/578) |
+
+## What is our deployment process for production?
+
+In preparation for our Tuesday and Thursday deployments, a cutoff time has been established.
+
+| Deployment Window | Cutoff Time |
+|:--|:--|
+| Tuesday, 10am EST | Monday, 12pm EST |
+| Thursday, 10am EST | Wednesday, 12pm EST |
+
+The cutoff time is automatically enforced via [this GitHub Action](.github/workflows/prepare_deployment_branch.yaml).
+
+1. At the specified cutoff a time, a PR is created to merge the contents of `master` to the `production` branch
```

Comment:
Other suggestion: prefer (in Markdown) to name all numbered list items with `1.` instead of 1, 2, 3, ... Markdown takes care of properly numbering things. It makes it so that if you insert things in the future (or remove them), you don't have to touch unrelated lines.

```suggestion
1. At the specified cut-off time (Mondays and Wednesdays at noon ET), the GitHub action creates a new branch named `deployment/YYYY-MM-DD` (where the YYYY-MM-DD is `today + 1day`, i.e. the date of the deployment, not the date of the branching) which branches from `master`. This branch now contains everything that was present in `master` at that cut-off time. This is the content that is/will be part of the production deployment.
1. A new PR from the deployment branch is filed to merge `deployment/YYYY-MM-DD` into `production`. The PR has title `"Deployment of YYYY-MM-DD"` and is tagged with the [`deployment` tag](https://github.com/CDCgov/prime-reportstream/issues?q=label%3Adeployment).
```

Reply:
Thanks - it helps to explain that this is both a PR and a new branch.